### PR TITLE
Revert changes to `publish-release` and `build-timestamped-master` workflows

### PR DESCRIPTION
**Pull request by sahanHe**
## Purpose

Fixes:[5768](https://github.com/ballerina-platform/ballerina-library/issues/5768)

## Checklist
- [x] Linked to an issue
- [ ] Updated the specification
- [ ] Updated the changelog
- [ ] Added tests
- [ ] Checked native-image compatibility

### CODE REVIEW

File: .github/workflows/build-timestamped-master.yml [hunk 1]
```diff
@@ -1,19 +1,64 @@
...
+          echo "version=${{ env.Initial_Version }}"
+          sed -i "s/version=\(.*\)/version=${{ env.Initial_Version }}/g" gradle.properties
+      - name: Publish to Nexus
+        env:
+          packageUser: ${{ secrets.BALLERINA_BOT_USERNAME }}
+          packagePAT: ${{ secrets.BALLERINA_BOT_TOKEN }}
+          nexusUser: ${{ secrets.NEXUS_USERNAME }}
+          nexusPassword: ${{ secrets.NEXUS_PASSWORD }}
+        run: |
+          ./gradlew clean publishMavenJavaPublicationToWSO2NexusRepository -x test --scan --no-daemon
```
Comment (daneshk) [inline .github/workflows/build-timestamped-master.yml:64]:
```suggestion
          ./gradlew clean publishMavenJavaPublicationToWSO2NexusRepository -x test --scan --no-daemon
         
```

File: .github/workflows/publish-release.yml [hunk 2]
```diff
@@ -1,16 +1,76 @@
...
+          REFRESH_TOKEN: ${{ secrets.REFRESH_TOKEN }}
+        run: |
+          ./gradlew clean release -Prelease.useAutomaticVersion=true
+          ./gradlew -Pversion=${VERSION} publish -x test -PpublishToCentral=true
+      - name: GitHub Release and Release Sync PR
+        env:
+          GITHUB_TOKEN: ${{ secrets.BALLERINA_BOT_TOKEN }}
+        run: |
+          gh release create v$VERSION --title "module-ballerinax-persist.sql-v$VERSION"
+          gh pr create --title "[Automated] Sync main after $VERSION release" --body "Sync main after $VERSION release"
```
Comment (daneshk) [inline .github/workflows/publish-release.yml:76]:
```suggestion
          gh pr create --title "[Automated] Sync main after $VERSION release" --body "Sync main after $VERSION release"
          
```

Comment (sahanHe) [reply .github/workflows/publish-release.yml]: updated in [ae7617b](https://github.com/ballerina-platform/module-ballerinax-persist.sql/pull/50/commits/ae7617b4bb11f97415ef5b514928ac45cd6fe0b5)

File: .github/workflows/build-timestamped-master.yml (hunk 1 above)
Comment (sahanHe) [reply .github/workflows/build-timestamped-master.yml]: updated in [ae7617b](https://github.com/ballerina-platform/module-ballerinax-persist.sql/pull/50/commits/ae7617b4bb11f97415ef5b514928ac45cd6fe0b5)

**Review by daneshk — APPROVED** [review]

**Pull request merged**